components/Layout.js

```diff
@@ -49,7 +66,7 @@ const NavLink = styled(Link)`
   &:active {
     background-color: #ffffff;
     color: #3498db;
-    font-weight: bold;
+    _font-weight: bold;
```

Comment:
If you don't need a property, I would just remove it.

Reply:
Just removed! :)